commons/src/main/kotlin/com/simplemobiletools/commons/extensions/Activity.kt

```diff
@@ -1194,7 +1194,7 @@ fun AppCompatActivity.showSideloadingDialog() {
 }
 
 fun BaseSimpleActivity.getTempFile(folderName: String, fileName: String): File? {
-    val folder = File(cacheDir, fileName)
+    val folder = File(cacheDir, folderName)
```

Comment:
why is this needed? Wont it break something?

Reply:
It would not break anything because of the way it's used. It was needed to [import blocked numbers](https://github.com/SimpleMobileTools/Simple-Commons/blob/master/commons/src/main/kotlin/com/simplemobiletools/commons/activities/ManageBlockedNumbersActivity.kt#L119) in the dialer app. Now, it would be used in the [Music app](https://github.com/KryptKode/Simple-Music-Player/blob/feat/edit-metadata/app/src/main/kotlin/com/simplemobiletools/musicplayer/helpers/TagHelper.kt#L41) to store a copy of music files temporarily when writing tags to them.

Reply:
There was an error in the method. It was using the `fileName` instead of the `folderName` so it's corrected here

Reply:
okay, lets hope